parachains/integration-tests/statemine/xcm/0_init.yml

```diff
@@ -98,28 +98,46 @@ tests:
               args: [
                 *ap_dest, # destination
                 {
-                  v2: [ #message
+                  v3: [ #message
+                    {
+                      UnpaidExecution: {
+                        weightLimit: Unlimited,
```

Comment:
Not sure if this will pass the Barrier, did it?

Reply:
yeah that got past the barrier (once the call was added to the safe call list)